### PR TITLE
Data Roles 2023

### PR DESCRIPTION
**Pull request by jraddon-octopus**
Re-aliging the Data Engineering roles after a year in review of them at Octopus.

### CODE REVIEW

**Review by ben-octo-data-dev — APPROVED** [review]
LGTM! :)

File: Data-Engineering/L3-Senior-Data-Engineer.md [hunk 1]
```diff
@@ -59,6 +59,7 @@
 
 - I performed regular interviews for _data-engineering_ candidates, and provided detailed and useful feedback.
 - I took on a significant share of unplanned work and other "housekeeping" tasks.
+- I put more technical concepts (e.g. data-models) into clear business language to facilitate the business outcomes and requirements expected for a project
```
Comment (ben-octo-data-dev) [inline Data-Engineering/L3-Senior-Data-Engineer.md:62]: Being super nit picky :) 

Missing a full-spot.

File: Data-Engineering/L4-Lead-Data-Engineer.md [hunk 2]
```diff
@@ -1,23 +1,24 @@
...
-- **Responsibility and direction needed**: Takes brand new ideas from inception to production with limited supervision.
+- **Responsibility and direction needed**: Can take an ambigious problem space and communicate insights that has strategic impact with limited supervision.
 
 ## 🦉 Domain expertise
 
-- I am a well-rounded data engineer and problem solver
-- I am an expert in my team's domain and various ETL solutions
+- I am a well-rounded data engineer and have strong analytical skills to solve and communicate problems
+- I am an expert in my team's domain I have a strong understanding of data-platform capabilities from ingestion through to visualisation
 - I am comfortable diving head-first into moderately risky situations with relatively little upfront information and oversight
```
Comment (ben-octo-data-dev) [inline Data-Engineering/L4-Lead-Data-Engineer.md:14]: Again, nit picky - full stops.

**Review by asmith-octopus — APPROVED** [review]

**Review by michaelnoonan — APPROVED** [review]
This is a nice set of clarifications given what I've seen demonstrated during 2023.

**Comment by Luckymhe** [comment]
Previous CLA is Pending? So never received assistance on it. No need to sign new cla

**Comment by Luckymhe** [comment]
Weather you signed or not you will never get assistance.

**Pull request merged**